### ISSUE TITLE
larastanStrictRules.listenerShouldHaveVoidReturnType detects middleware as listener, because it has only handle method

### ISSUE THREAD

**Issue by webard** [canvural/larastan-strict-rules#211]
```
Listeners handle method should have 'void' return type.   
         🪪  larastanStrictRules.listenerShouldHaveVoidReturnType  
         ✏️  app/Http/Middleware/ResolveActivityLogName.php:19  
```

**Comment by canvural** [comment]
Fixed on `3.0.x` in 06eaa05 (via #220): Laravel commands and middlewares are no longer reported by `listenerShouldHaveVoidReturnType`. Commands are matched with an `instanceof` check, middlewares by their `handle(Request $request, Closure $next)` signature, since they have no contract to check against.

Leaving this open until 3.0 is released, as the currently released 2.2.x line still has the false positive.

**Comment by canvural** [comment]
Fixed on `3.0.x` in 06eaa05 — this ships in 3.0.4.
